homeassistant/components/zwave/node_entity.py

```diff
@@ -107,13 +108,20 @@ def __init__(self, node, network, new_entity_ids):
         dispatcher.connect(
             self.network_scene_activated, ZWaveNetwork.SIGNAL_SCENE_EVENT)
 
-    def network_node_changed(self, node=None, args=None):
+    def network_node_changed(self, node=None, value=None, args=None):
         """Handle a changed node on the network."""
         if node and node.node_id != self.node_id:
             return
         if args is not None and 'nodeId' in args and \
                 args['nodeId'] != self.node_id:
             return
+
+        # Process central scene activation
+        if (value is not None and
+                value.command_class == COMMAND_CLASS_CENTRAL_SCENE):
+            self.central_scene_activated(value.index, value.data)
+            return
```

Comment:
I think we should get rid of this return, and let the node statistics also update after this message is received.